EDDI/MainWindow.xaml.cs

```diff
@@ -604,6 +605,7 @@ public static void prepareLog(IProgress<string> progress)
                         string formatString = "s"; // i.e. DateTimeFormatInfo.SortableDateTimePattern
                         if (DateTime.TryParseExact(linedatestring, formatString, CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal, out DateTime linedate))
                         {
+                            linedate = linedate.ToUniversalTime();
```

Comment:
Aha!

On closer reading of the docs, passing the DateTimeStyles.AssumeUniversal _and_ DateTimeStyles.AdjustToUniversal flags to TryParseExact() would have the same effect, whereas DateTimeStyles.AssumeUniversal on its own converts 
to local time.

I was under the misconception that DataTime structs represented an absolute point in time and that the time zone only affected output formatting.